src/neo-vm/ExecutionEngine.cs

```diff
@@ -1443,6 +1463,8 @@ protected virtual void PostExecuteInstruction(Instruction instruction)
         {
             if (ReferenceCounter.CheckZeroReferred() > MaxStackSize)
                 throw new InvalidOperationException($"MaxStackSize exceed: {ReferenceCounter.Count}");
+            if (!ipFlag) CurrentContext?.MoveNext();
```

Comment:
Can we move this to line 1250? otherwise we can override and break the logic if we don't call `base.PostExecuteInstruction` in order to avoid limits.

Reply:
To avoid limits, you should override `MaxStackSize`.